database/migrations/2022_07_08_145847_create_spam_table.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up()
+    {
+        Schema::create('spam', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedInteger('user_id')->nullable();
```

Comment:
This `user_id` should not be nullable I think?

Reply:
resolved. Before I was thinking to allow a flexible development although on second thought, I can't think of any use cases where a user_id may be null. https://github.com/laravelio/laravel.io/pull/886/commits/8fd3e13c4f6cb5e1b6cdf12f8ff35274636cfa01